docs/root/configuration/observability/access_log/usage.rst

```diff
@@ -579,7 +579,10 @@ The following command operators are supported:
   multiple instances or between restarts.
 
 %GRPC_STATUS%
-  gRPC status code which is easy to interpret with text message corresponding with number.
+  enum string(camel case, like InvalidArgument) of `gRPC status code <https://github.com/googleapis/googleapis/blob/master/google/rpc/code.proto>`.
+
+%GRPC_STATUS_SNAKE_STRING%
```

Comment:
I think it would be better if you modify existing GRPC_STATUS to take optional parameter. If not specified, it would be camel case. Params would be CAMEL or SNAKE.

Reply:
DONE.